drivers/volume/portworx/portworx.go

```diff
@@ -450,6 +455,25 @@ func (d *portworx) init(sched, nodeDriver, token, storageProvisioner, csiGeneric
 	} else {
 		torpedovolume.StorageProvisioner = provisioners[torpedovolume.DefaultStorageProvisioner]
 	}
+
+	// Update node PxPodRestartCount during init
+	k8s, err := scheduler.Get(sched)
```

Comment:
How about? or something else instead of k8s

```
schedDriver, err := scheduler.Get(sched)
```

Reply:
Done